src/lib/helpers/test/setupTestEnv.ts

```diff
@@ -39,13 +42,38 @@ function setupTestEnv() {
   };
 }
 
-// Function to check if the given chain ID is valid and return it as VALID_CHAIN_IDS
+/**
+ * Validates the provided chain ID against a list of valid chain IDs.
+ * @param {number} chainId - The chain ID to validate.
+ * @returns {VALID_CHAIN_IDS} - The validated chain ID.
+ * @throws {Error} If the chain ID is not valid.
+ */
 function getValidChainId(chainId: number): VALID_CHAIN_IDS {
   if (chainId === 11155111 || chainId === 1) {
     return chainId as VALID_CHAIN_IDS;
   }
   throw new Error(`Invalid chain ID: ${chainId}`);
 }
 
-export { getValidChainId };
+/**
+ * Retrieves the RPC URL based on the `useLocal` flag.
+ * @param {boolean} useLocal - Determines which environment variable to use for the RPC URL.
+ * @returns {string} The RPC URL.
+ * @throws {Error} If the corresponding environment variable is not defined.
+ */
+function getRpcUrl(useLocal = false) {
+  const rpcUrl = useLocal
+    ? process.env.TEST_LOCAL_NODE_RPC_URL
```

Comment:
Let's add this to `.env.example`

Reply:
https://github.com/ScopeLift/stealth-address-sdk/pull/26/commits/efe0eb52edc106c60a0667646a9029f29e4b74e3